### ISSUE TITLE
Feature Request: Ability to mark steps as 'pending'

### ISSUE THREAD

**Issue by tknerr** [radish-bdd/radish#18]
So far we can only raise an error if a step is not implemented yet:
```python
@step(re.compile(r"some step to be implemented"))
def my_step(step):
    raise NotImplementedError("This step is not implemented yet")
```

However, this causes the whole test run to fail if we run with `radish -e`

It would be super useful to mark a step as pending.

See here:
https://github.com/cucumber/cucumber/wiki/Step-Definitions#pending-steps

**Comment by timofurrer** [comment]
Implemented with commit 9b4eb02

**Comment by tknerr** [comment]
Thanks @timofurrer !
